src/resources/ha-style.js

```diff
@@ -52,6 +53,7 @@ documentContainer.innerHTML = `<custom-style>
       --card-background-color: #ffffff;
       --primary-background-color: #fafafa;
       --secondary-background-color: #e5e5e5; /* behind the cards on state */
+      --dark-background-color: #455a64;
```

Comment:
I think that we should move this to the main interface colors as the `--dark-color`. 

Actually, I think that we should not add this to ha-style at all. Let's keep it inside the hui-root for now. You can specify defaults so that users can override it if they want.

```
--app-toolbar-background: var(--dark-color, #455a64);
```

If the user has specified dark-color, they can override it. But I don't want people to think that this is some official theme variable.